static/app/components/dashboards/widgetQueriesForm.tsx

```diff
@@ -50,6 +50,8 @@ type Props = {
  * callback. This component's state should live in the parent.
  */
 class WidgetQueriesForm extends React.Component<Props> {
+  blurTimeout: ReturnType<typeof setTimeout> | undefined = undefined;
```

Comment:
I would use `null` as the sentinel value here.

Here's a nice article: https://dev.to/bhushanshahdev/undefined-vs-null-575e

Reply:
Nice article, updated!